	Declare all functions used in only one file as static.

diff -ur dvipng-1.12.orig/color.c dvipng-1.12/color.c
--- dvipng-1.12.orig/color.c	2009-03-25 14:06:21.000000000 +0100
+++ dvipng-1.12/color.c	2009-06-09 12:58:51.000000000 +0200
@@ -54,7 +54,7 @@
    cstack[1].blue=0; 
 }
 
-struct colorname * NewColor(char* prefix, int nprefix,
+static struct colorname * NewColor(char* prefix, int nprefix,
               char* name, int nname,
 	      char* model, int nmodel,
 	      char* values, int nvalues)
@@ -92,7 +92,7 @@
 #define FINDPSNAMEEND(s,n) n=0; while(s<max && *s!='{') { s++; n++; }
 #define BLANKCOMMAS(s) 
 
-struct colorname* LoadColornameFile(char* filename)
+static struct colorname* LoadColornameFile(char* filename)
 {
   struct colorname *list=NULL,*tmp=NULL; 
   char *filepath,*pos,*max;
@@ -165,7 +165,7 @@
   return(list);
 }
 
-void ClearXColorPrologue(void)
+static void ClearXColorPrologue(void)
 {
   struct colorname *next;
   while (xcp) {
@@ -199,7 +199,7 @@
   strcpy(xcpname,name);
 }
 
-struct colorname* LoadXColorPrologue(void)
+static struct colorname* LoadXColorPrologue(void)
 {
   struct colorname *list=NULL,*tmp=NULL; 
   char *filepath,*pos,*max;
diff -ur dvipng-1.12.orig/draw.c dvipng-1.12/draw.c
--- dvipng-1.12.orig/draw.c	2008-05-05 22:54:51.000000000 +0200
+++ dvipng-1.12/draw.c	2009-06-09 12:37:32.000000000 +0200
@@ -284,7 +284,7 @@
 }
 
 
-void DrawPage(dviunits hoffset, dviunits voffset) 
+static void DrawPage(dviunits hoffset, dviunits voffset) 
      /* To be used after having read BOP and will exit cleanly when
       * encountering EOP.
       */
diff -ur dvipng-1.12.orig/dvi.c dvipng-1.12/dvi.c
--- dvipng-1.12.orig/dvi.c	2008-05-05 22:54:51.000000000 +0200
+++ dvipng-1.12/dvi.c	2009-06-09 12:36:50.000000000 +0200
@@ -56,7 +56,7 @@
   return (unsigned char)got;
 }
 
-void DVIInit(struct dvi_data* dvi)
+static void DVIInit(struct dvi_data* dvi)
 {
   int     k;
   unsigned char* pre;
@@ -277,7 +277,7 @@
   return(length);
 }
 
-void SkipPage(struct dvi_data* dvi)
+static void SkipPage(struct dvi_data* dvi)
 { 
   /* Skip present page */
   unsigned char* command;           
@@ -307,7 +307,7 @@
   DEBUG_PRINT(DEBUG_DVI,("SKIP CMD:\t%s", dvi_commands[*command]));
 }
 
-struct page_list* InitPage(struct dvi_data* dvi)
+static struct page_list* InitPage(struct dvi_data* dvi)
 {
   /* Find page start, return pointer to page_list entry if found */
   struct page_list* tpagelistp=NULL;
@@ -428,7 +428,7 @@
 }
 
 
-void DelPageList(struct dvi_data* dvi)
+static void DelPageList(struct dvi_data* dvi)
 {
   struct page_list* temp;
   
diff -ur dvipng-1.12.orig/enc.c dvipng-1.12/enc.c
--- dvipng-1.12.orig/enc.c	2008-05-05 20:58:00.000000000 +0200
+++ dvipng-1.12/enc.c	2009-06-09 12:53:02.000000000 +0200
@@ -26,7 +26,7 @@
 
 struct encoding* encodingp=NULL;
 
-struct encoding* InitEncoding(char* encoding) 
+static struct encoding* InitEncoding(char* encoding) 
 {
   char *pos,*max,*buf,*enc_file;
   int i;
diff -ur dvipng-1.12.orig/font.c dvipng-1.12/font.c
--- dvipng-1.12.orig/font.c	2008-02-08 14:45:24.000000000 +0100
+++ dvipng-1.12/font.c	2009-06-09 12:40:10.000000000 +0200
@@ -38,7 +38,7 @@
 }
 
 
-double ActualFactor(uint32_t unmodsize)
+static double ActualFactor(uint32_t unmodsize)
 /* compute the actual size factor given the approximation */
 /* actually factor * 1000 */
 {
@@ -189,7 +189,7 @@
 }
 
 #ifdef HAVE_FT2_OR_LIBT1
-char* kpse_find_t1_or_tt(char* filename) 
+static char* kpse_find_t1_or_tt(char* filename) 
 {
     char* filepath = kpse_find_file(filename, kpse_type1_format, false);
 #ifdef HAVE_FT2
@@ -200,7 +200,7 @@
 }
 #endif
 
-void FontFind(struct font_entry * tfontptr)
+static void FontFind(struct font_entry * tfontptr)
 {
 #ifdef HAVE_LIBKPATHSEA
   kpse_glyph_file_type font_ret;
@@ -300,7 +300,7 @@
 }
 
 
-void DoneFont(struct font_entry *tfontp)
+static void DoneFont(struct font_entry *tfontp)
 {
   switch (tfontp->type) {
   case FONT_TYPE_PK:
diff -ur dvipng-1.12.orig/fontmap.c dvipng-1.12/fontmap.c
--- dvipng-1.12.orig/fontmap.c	2009-04-22 14:14:06.000000000 +0200
+++ dvipng-1.12/fontmap.c	2009-06-09 12:55:27.000000000 +0200
@@ -30,7 +30,7 @@
 #endif
 static struct psfontmap *psfontmap=NULL;
 
-char* newword(char** buffer, char* end) 
+static char* newword(char** buffer, char* end) 
 {
   char *word,*pos=*buffer;
 
@@ -55,7 +55,7 @@
   return(word);
 }
 
-char* find_format(char* name)
+static char* find_format(char* name)
 {
   /* Cater for both new (first case) and old (second case) kpathsea */
   char* format =
@@ -134,7 +134,7 @@
   return(newentry);
 }
 
-struct psfontmap *SearchPSFontMap(char* fontname,
+static struct psfontmap *SearchPSFontMap(char* fontname,
 				  struct filemmap* search_mmap)
 {
   static char *pos=NULL,*end=NULL;
@@ -196,7 +196,7 @@
 #endif
 }
 
-void ReadPSFontMap(struct psfontmap *entry)
+static void ReadPSFontMap(struct psfontmap *entry)
 {
   char *pos,*end,*psname;
   int nameno = 0;
diff -ur dvipng-1.12.orig/ft.c dvipng-1.12/ft.c
--- dvipng-1.12.orig/ft.c	2009-03-25 14:04:41.000000000 +0100
+++ dvipng-1.12/ft.c	2009-06-09 12:56:39.000000000 +0200
@@ -145,7 +145,7 @@
 }
 
 
-void UnLoadFT(struct char_entry *ptr)
+static void UnLoadFT(struct char_entry *ptr)
 {
   if (ptr->data!=NULL)
     free(ptr->data);
diff -ur dvipng-1.12.orig/papersiz.c dvipng-1.12/papersiz.c
--- dvipng-1.12.orig/papersiz.c	2008-02-08 14:45:24.000000000 +0100
+++ dvipng-1.12/papersiz.c	2009-06-09 12:59:23.000000000 +0200
@@ -36,7 +36,7 @@
  *   Advance the passed pointer as well.
  */
 
-int32_t myatodim(char ** p)
+static int32_t myatodim(char ** p)
 { 
   double tmp; /* double accuracy is enough, I think */
   int i=0;
diff -ur dvipng-1.12.orig/pk.c dvipng-1.12/pk.c
--- dvipng-1.12.orig/pk.c	2009-03-25 14:05:54.000000000 +0100
+++ dvipng-1.12/pk.c	2009-06-09 12:47:48.000000000 +0200
@@ -35,7 +35,7 @@
 int             repeatcount;
 int             poshalf;
 
-unsigned char getnyb(unsigned char** pos)
+static unsigned char getnyb(unsigned char** pos)
 {
   if (poshalf == 0) {
     poshalf=1;
@@ -46,7 +46,7 @@
   }
 }
 
-uint32_t pk_packed_num(unsigned char** pos)
+static uint32_t pk_packed_num(unsigned char** pos)
 {
   register int    i;
   uint32_t        j;
@@ -77,7 +77,7 @@
   }
 }
 
-unsigned char* skip_specials(unsigned char* pos)
+static unsigned char* skip_specials(unsigned char* pos)
 {
   uint32_t    i;
 
@@ -384,7 +384,7 @@
   }
 }
 
-void UnLoadPK(struct char_entry *ptr)
+static void UnLoadPK(struct char_entry *ptr)
 {
   if (ptr->data!=NULL)
     free(ptr->data);
diff -ur dvipng-1.12.orig/ppagelist.c dvipng-1.12/ppagelist.c
--- dvipng-1.12.orig/ppagelist.c	2008-02-08 14:45:23.000000000 +0100
+++ dvipng-1.12/ppagelist.c	2009-06-09 12:45:06.000000000 +0200
@@ -107,7 +107,7 @@
   return(false);
 }
 
-void ListPage(int32_t pslow, int32_t pshigh)
+static void ListPage(int32_t pslow, int32_t pshigh)
 {
   register struct pp_list   *pl;
 
diff -ur dvipng-1.12.orig/set.c dvipng-1.12/set.c
--- dvipng-1.12.orig/set.c	2009-03-25 14:06:04.000000000 +0100
+++ dvipng-1.12/set.c	2009-06-09 12:45:07.000000000 +0200
@@ -84,7 +84,7 @@
 }
 
 
-void ChangeColor(gdImagePtr imagep,int x1,int y1,
+static void ChangeColor(gdImagePtr imagep,int x1,int y1,
 		 int x2,int y2,int color1,int color2)
 /* In the given rectangle, change color1 to color2 */
 {
diff -ur dvipng-1.12.orig/sfd.c dvipng-1.12/sfd.c
--- dvipng-1.12.orig/sfd.c	2008-05-05 20:58:00.000000000 +0200
+++ dvipng-1.12/sfd.c	2009-06-09 12:49:29.000000000 +0200
@@ -25,7 +25,7 @@
 #include "dvipng.h"
 struct subfont* subfontp=NULL;
 
-struct subfont* ReadSubfont(char* sfdname, char *infix) 
+static struct subfont* ReadSubfont(char* sfdname, char *infix) 
 {
   char *pos,*max,*sfdfile=NULL;
   struct subfont* sfdp=NULL;
diff -ur dvipng-1.12.orig/special.c dvipng-1.12/special.c
--- dvipng-1.12.orig/special.c	2009-03-24 15:03:27.000000000 +0100
+++ dvipng-1.12/special.c	2009-06-09 13:01:45.000000000 +0200
@@ -40,7 +40,7 @@
 
 struct pscode* psheaderp=NULL; /* static, DVI-specific header list */
 
-void PSCodeInit(struct pscode *entry, char *special)
+static void PSCodeInit(struct pscode *entry, char *special)
 {
   entry->next=NULL;
   entry->special=special;
@@ -91,7 +91,7 @@
   }
 }
 
-void writepscode(struct pscode* pscodep, FILE* psstream)
+static void writepscode(struct pscode* pscodep, FILE* psstream)
 {
   while (pscodep!=NULL) {
     if (pscodep->code!=NULL) {
@@ -131,7 +131,7 @@
 }
 
 
-gdImagePtr
+static gdImagePtr
 ps2png(struct pscode* pscodep, char *device, int hresolution, int vresolution, 
        int llx, int lly, int urx, int ury, int bgred, int bggreen, int bgblue)
 {
@@ -274,7 +274,7 @@
   return psimage;
 }
 
-gdImagePtr
+static gdImagePtr
 rescale(gdImagePtr psimage, int pngwidth, int pngheight)
 {
   gdImagePtr scaledimage=psimage;
@@ -305,7 +305,7 @@
   return(scaledimage);
 }
 
-void newpsheader(char* special) {
+static void newpsheader(char* special) {
   struct pscode* tmp;
   char* txt;
 
diff -ur dvipng-1.12.orig/t1.c dvipng-1.12/t1.c
--- dvipng-1.12.orig/t1.c	2009-03-25 14:06:12.000000000 +0100
+++ dvipng-1.12/t1.c	2009-06-09 12:56:08.000000000 +0200
@@ -164,7 +164,7 @@
 }
 
 
-void UnLoadT1(struct char_entry *ptr)
+static void UnLoadT1(struct char_entry *ptr)
 {
   if (ptr->data!=NULL)
     free(ptr->data);
